modules/instrument_builder/jsx/react.elements.js

```diff
@@ -0,0 +1,189 @@
+HeaderElement = React.createClass({
+	render: function(){
+		return (
+			<h2>{this.props.header}</h2>
+		)
+	}
+});
+
+LabelElement = React.createClass({
+	render: function(){
+		return (
+			<p>{this.props.label}</p>
+		)
+	}
+});
+
+ScoredElement = React.createClass({
+	render: function(){
+		return (
+			<div>
+			<label className="lab col-sm-4 col-xs-12">
+				{this.props.label}
+			</label>
+			<div className="col-sm-8">
+				<div className="col-xs-12 element">
+					0
+				</div>
+			</div>
+			</div>
+		)
+	}
+});
+
+TextboxElement = React.createClass({
+	render: function(){
+		return (
+			<div>
+			<label className="lab col-sm-4 col-xs-12">
+				{this.props.label}
+			</label>
+			<div className="col-sm-8">
+				<div className="col-xs-12 col-sm-6 element">
+					<input type="text" className="form-control input-sm user-success" />
+				</div>
+			</div>
+			</div>
+		)
+	}
+});
+
+TextareaElement = React.createClass({
+	render: function(){
+		return (
+			<div>
+			<label className="lab col-sm-4 col-xs-12">
+				{this.props.label}
+			</label>
+			<div className="col-sm-8">
+				<div className="col-xs-12 col-sm-6 element">
+					<textarea cols="25" rows="4" className="form-control input-sm user-success"></textarea>
+				</div>
+			</div>
+			</div>
+		)
+	}
+});
+
+SelectElement = React.createClass({
+	render: function(){
+		var multiple = '';
+		if(this.props.multiple){
+			multiple = 'multiple';
+		}
+		return (
+			<div>
+			<label className="lab col-sm-4 col-xs-12">
+				{this.props.label}
+			</label>
+			<div className="col-sm-8">
+				<div className="col-xs-12 col-sm-6 element">
+					<select multiple={multiple} className="form-control input-sm">
+						{Object.keys(this.props.options).map(function(option){
+							return (
+								<option>
+									{option}
+								</option>
+							)
+						})}
+					</select>
+				</div>
+			</div>
+			</div>
+		)
+	}
+});
+
+DateElement = React.createClass({
+	render: function(){
+		var max = this.props.maxYear,
+			min = this.props.minYear;
```

Comment:
shouldn't these have the date part added to them? Otherwise, what's the point of putting them in a variable instead of just directly using the prop in the render method?